### ISSUE TITLE
Lucifer cannon power warning noise doesn't sound while jetpack on

### ISSUE THREAD

**Issue by overvolting** [Unvanquished/Unvanquished#579]
Flying with jetpack and powering up lucifer cannon doesn't produce the lucifer cannon overload warning.

**Comment by overvolting** [comment]
I would like to add that when you're flying with the jetpack many other sounds also don't play.

For example, the sound of an active saw.

**Comment by DolceTriade** [comment]
Multiple sounds on the same channel?

**Comment by Kangz** [comment]
Yes it is a limitation of the q3 sound model: you can only have one looping sound per entity. I want to move away from this eventually.

**Comment by Viech** [comment]
In that case I won't bother working around this now.

**Comment by bmorel** [comment]
> Flying with jetpack and powering up lucifer cannon doesn't produce the lucifer cannon overload warning

This actually seems fixed now (in 0.52.1) since the jetpack sound is cut to let the luci warn, but not that:

> I would like to add that when you're flying with the jetpack many other sounds also don't play. For example, the sound of an active saw.

although I only have the problem with either luci warning or painsaw, no other weapon seems affected.

**Comment by bmorel** [comment]
> although I only have the problem with either luci warning or painsaw, no other weapon seems affected.

Correction, this also happens with ambient sounds, for example in plat23. This does not, however, happens with buildings sounds like turrets "scan blips" or RC humming.
